djsixpack/djsixpack.py

```diff
@@ -5,6 +5,8 @@
 from django.conf import settings
 from requests.exceptions import RequestException
 
+from .models import SixpackParticipant
```

Comment:
As per "Writing Idiomatic Python", we prefer using absolute imports to relative imports.